### ISSUE TITLE
thunderbird: define MOZ_USE_XINPUT2 globally

### ISSUE THREAD

**Issue by jyhi** [AOSC-Dev/aosc-os-abbs#2096]
Bug description
-----------------------

a23cabe0 defines environment variable `MOZ_USE_XINPUT2=1` which tells Firefox to support touch scrolling and multi-touch:

- https://wiki.gentoo.org/wiki/Firefox#Enabling_multitouch

This works fine for me, but in scenarios when the `firefox` binary is directly called, Firefox will still use XInput 1 and simulates mouse scrolling upon touchpad scrolling events. This causes UX inconsistency.

Reproducing steps
---------------------------

1. Open Firefox using the desktop definition to use XInput 2.
2. Close Firefox, then directly invoke `firefox` to use XInput 1.

Update channel(s)
--------------------------

I'm on Stable, but I suppose all channels are affected.

Package and version
------------------------------

```
$ dpkg -s firefox
Package: firefox
Status: install ok installed
Section: web
Installed-Size: 224652
Maintainer: Mingcong Bai <jeffbai@aosc.io>
Architecture: amd64
Version: 72.0.2
Replaces: firefox-i18n (<= 43.0.1)
Depends: gtk-2 (>= 2.24.32), startup-notification (>= 0.12-2), mime-types (>= 9-1), dbus-glib (>= 0.110), alsa-lib (>= 1.2.1.2), desktop-file-utils (>= 0.24), hicolor-icon-theme (>= 0.17), libevent (>= 2.1.11), nss (>= 3.47.1), hunspell (>= 1.7.0-1), sqlite (>= 3.30.1), networkmanager (>= 1.18.0), mozilla-common (>= 1.4-2), ffmpeg (>= 4.2.1-1), icu (>= 64.2-2), gtk-3 (>= 3.24.12)
Breaks: firefox-i18n (<= 43.0.1)
Description: Standalone Web browser from Mozilla
```

**Comment by jyhi** [comment]
One concern (and it's also Firefox developers' concern) is that they find this buggy:

- https://bugzilla.mozilla.org/show_bug.cgi?id=1390795: "We explicitly keep MOZ_USE_XINPUT2 off by default because behaviour is buggy when it is turned on."
- https://bugzilla.mozilla.org/show_bug.cgi?id=1182700: (Open) Wheel scrolling periodically stops working with XInput2
- https://bugzilla.mozilla.org/show_bug.cgi?id=1170342: (Closed) Focus out events don't get sent sometimes on GTK3 with XInput2, causing tests to hang

I personally haven't experienced any bug regarding to this.

**Comment by jyhi** [comment]
Thunderbird seems to be able to benefit from this too. I tested with `env MOZ_USE_XINPUT2=1 thunderbird` and the scrolling experience is obviously different, except that there is still no inertial scrolling.

**Comment by MingcongBai** [comment]
It seems that Firefox enables this behaviour by default, and Thunderbird is still left out.

**Comment by MingcongBai** [comment]
Addressed with #2882. Closing.
